hudi-cli/src/main/java/org/apache/hudi/cli/commands/BootstrapCommand.java

```diff
@@ -72,8 +72,12 @@ public String bootstrap(
           help = "Class for Full bootstrap input provider") final String fullBootstrapInputProvider,
       @ShellOption(value = {"--schemaProviderClass"}, defaultValue = "",
           help = "SchemaProvider to attach schemas to bootstrap source data") final String schemaProviderClass,
-      @ShellOption(value = {"--payloadClass"}, defaultValue = "org.apache.hudi.common.model.OverwriteWithLatestAvroPayload",
-          help = "Payload Class") final String payloadClass,
+      @ShellOption(value = {"--payloadClass"}, defaultValue = "",
+          help = "Deprecated. Use merge-mode for overwrite or event time merging. Payload Class") final String payloadClass,
```

Comment:
```suggestion
          help = "Payload Class (deprecated). Use merge-mode for overwrite or event time merging.") final String payloadClass,
```
Also add more details on how `CUSTOM` merge mode can be used.